core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -3940,6 +3940,12 @@ groups:
     - release-openshift-origin-installer-e2e-aws-upgrade-4.7-stable-to-4.8-ci
     - release-openshift-origin-installer-e2e-aws-upgrade-rollback-4.7-to-4.8
     - release-openshift-origin-installer-e2e-aws-upgrade-rollback-4.8
+    - release-openshift-origin-installer-e2e-remote-libvirt-ppc64le-4.8
+    - release-openshift-origin-installer-e2e-remote-libvirt-s390x-4.8
+    - release-openshift-origin-installer-e2e-remote-libvirt-image-ecosystem-s390x-4.8
+    - release-openshift-origin-installer-e2e-remote-libvirt-image-ecosystem-ppc64le-4.8
+    - release-openshift-origin-installer-e2e-remote-libvirt-jenkins-e2e-s390x-4.8
+    - release-openshift-origin-installer-e2e-remote-libvirt-jenkins-e2e-ppc64le-4.8
```

Comment:
Do these *need* to be on api.ci? We want to decomission that cluster and if possible, no new jobs should be added on it.

Reply:
We have a PR in place for the same https://github.com/openshift/release/pull/16184